### PR TITLE
Rbrauer braai food

### PR DESCRIPTION
**Pull request by rbrauer007**
### Description of Change

Added some food!

### Test results

Tests ... I don't test; my sh!t always works first time!

### Who should know about the change?

/cc: @githubteacher 
/cc: @GlenWatridge 
/cc: @ixmarr 
/cc @ChrissieG9

### CODE REVIEW

File: braai-food.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Food
+
+- Boerewors
+- Lamb Chops
```
Comment (GlenWatridge) [inline braai-food.md:4]: @rbrauer007 Don't eat lamb chops

**Review by GlenWatridge — COMMENTED** [review]
Updated

**Review by nagsmandya — APPROVED** [review]
Done :+1:

**Review by GlenWatridge — APPROVED** [review]
oooooooooooooookay Approved

**Pull request merged**